source/extensions/common/wasm/null/null.h

```diff
@@ -15,7 +15,7 @@ using FilterHeadersStatus = Http::FilterHeadersStatus;
 using FilterMetadataStatus = Http::FilterMetadataStatus;
 using FilterTrailersStatus = Http::FilterTrailersStatus;
 using FilterDataStatus = Http::FilterDataStatus;
-using GrpcStatus = ProtobufUtil::error::Code;
+using GrpcStatus = Envoy::Grpc::Status::GrpcStatus;
```

Comment:
Naive question: why?

Reply:
Because the latter defines an invalid status which I can use to indicate that the gRPC status is not provided because the envoy function takes std::optional<GrpcStatus> but the ABI is based on raw words, so std::optional is not available and I need a sentinel value.